OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -114,6 +127,22 @@ function Get-WorkerPoolMembership
     return $workerPoolMembership
 }
 
+Function Get-Space
+{
+    # Define parameters
+    param(
+        $SpaceName,
+        $ServerUrl,
+        $APIKey
+    )
+
+    # Get all spaces
```

Comment:
These comments are a bit redundant. I go by the theory that if I need a comment, then I've failed in making the code understandable - the code should be easy to reason about without the comments. It's soo easy for comments to get out of date.